jerry-core/ecma/builtin-objects/ecma-builtin-array.inc.h

```diff
@@ -19,6 +19,8 @@
 
 #include "ecma-builtin-helpers-macro-defines.inc.h"
 
+#ifndef CONFIG_DISABLE_ARRAY_BUILTIN
+
```

Comment:
There is no need for anything special. When you check a .inc file, you can grep the file which includes it, and check the first condition in the beginning of that file. Btw you can throw an error if multiple file includes an built-in inc file.